roles/openshift_metrics/templates/heapster.j2

```diff
@@ -0,0 +1,98 @@
+apiVersion: "v1"
+kind: "ReplicationController"
+metadata:
+  name: heapster
+  labels:
+    metrics-infra: heapster
+    name: heapster
+spec:
+  selector:
+    name: heapster
+  replicas: 0
```

Comment:
@ewolinetz  thoughts?  do we see this as an issue if we are scaling down and then scaling up anyway?